integration_tests/steps/integration_tests.py

```diff
@@ -390,6 +390,11 @@ def test_export_data_csv(context, scenario):
                                                                     file=data_fn))
 
 
+@then('we get the area tree summary')
+def area_tree(context):
+    get_simulation_raw_results(context)
```

Comment:
If this is not asserting anything and it's just performing some operations, I'd suggest using a `@when` step (and using it in the `.feature` files as `and`).

Reply:
What happens if this step is not performed? If all `test_export_of_kpi_result` must necessarily use this step, I think it would make sense to directly add it inside of `test_export_of_kpi_result`. Please let me know if I'm missing something!

Reply:
I agree with Pierpaolo. 
Please add these lines to test_export_of_kpi_result
```
    area_tree_summary = glob.glob(os.path.join(context.export_path, "*", "area_tree_summary.json"))
    with open(area_tree_summary[0], "r") as sf:
        context.area_tree_summary_data = json.load(sf)
```